DirectProgramming/DPC++FPGA/Tutorials/Features/max_interleaving/src/max_interleaving.cpp

```diff
@@ -0,0 +1,202 @@
+//==============================================================
+// Copyright Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+// =============================================================
+#include <CL/sycl.hpp>
+#include <array>
+#include <iomanip>
+#include <iostream>
+#include "dpc_common.hpp"
```

Comment:
Please add the comments below for for the include of dpc_common.hp. it should look like this:

// dpc_common.hpp can be found in the dev-utilities include folder.
// e.g., $ONEAPI_ROOT/dev-utilities/<version>/include/dpc_common.hpp
#include "dpc_common.hpp"

as seen in https://github.com/oneapi-src/oneAPI-samples/blob/master/DirectProgramming/DPC%2B%2B/CombinationalLogic/mandelbrot/src/main.cpp

Reply:
Thanks for the suggestions, I have updated the source code accordingly.